partiql-ast/src/main/kotlin/org/partiql/ast/normalize/Normalize.kt

```diff
@@ -19,7 +19,7 @@ import org.partiql.ast.Statement
 /**
  * AST normalization
  */
-public fun Statement.normalize(): Statement {
+public fun Statement.normalize(): Statement { // TODO: Make this Java friendly and consider moving to planner package.
```

Comment:
I _somewhat_ think it belongs here? . . but am not so sure.

I also wouldn't say it's Java "unfriendly" considering it's "NormalizeKt.normalize(ast)" .. but perhaps we simply add the @JvmName("Normalize") to the top.

Another option is to add a "normalize" method to each AST node which accepts a "Normalizer" interface for things like producing synthetic bindings .. or is that just another visitor!